metatron-app/metatron-discovery
# Fix token extension when check_token is called

### Description

Fix token extension when check_token is called

**Related Issue** : 



### How Has This Been Tested?

1. Login to the external client using metatron Discovery.
2. The accessToken is issued using the parameter(code) received from redirectUri.
3. Call **'/oauth/check_token'** using accessToken and check if the accessToken expiration time is extended.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
